src/util.ts

```diff
@@ -87,6 +87,7 @@ export async function showWorkspaceFoldersQuickPick(placeHolderString: string):
     const pickedItem = await vscode.window.showQuickPick(folderQuickPickItems, folderQuickPickOption);
 
     if (!pickedItem) {
+        telemetryProperties.cancelStep = placeHolderString;
```

Comment:
nit: I'd prefer if this was just a constant string like "showWorkspaceFolders". We're never going to have _multiple_ calls to showWorkspaceFolders in one command so I'd rather have something consistent across all commands